clusterloader2/testing/load/golang/custom_api_call_thresholds.yaml

```diff
@@ -0,0 +1,11 @@
+CUSTOM_API_CALL_THRESHOLDS: |
```

Comment:
Why this was backported?

Reply:
I've backported it (whole custom thresholds) to not have merge conflicts, but it won't be used in our tests afaik